Don't ignore prefix length from input stream when -t and -p are used together.
From FreeBSD PR ports/131259

--- aggregate.c.orig	Wed Mar  6 16:59:37 2002
+++ aggregate.c	Thu May  3 15:08:40 2012
@@ -261,8 +261,18 @@ int main(int argc, char **argv)
           moanf(0, "[line %d] line too long; ignoring line", line);
         continue;
       }
+      if (buf[i] == '/') { 
+           masklen = atoi(buf + i + 1);
+         if (masklen < 1 || masklen > max_prefix_length)
+        {
+              if (!quiet)
+            moanf(0, "[line %d] mask length %d out of range; ignoring line", \
+                   line, masklen);
+          continue;
+          }
+      } else
+             masklen = default_prefix_length;
       buf[i] = 0;
-      masklen = default_prefix_length;
     } else {
       i = 0;
       while (buf[i] != '/' && i < MAX_buf) i++;
